### PR TITLE
feat: Camunda Modeler 5 compatibility patch

### LINKED ISSUE #17: Migrate to latest properties panel version + plugin helpers
__What should we do?__

* Update to use components over elements (cf. https://github.com/bpmn-io/bpmn-js-examples/issues/160)
* Update to mark plugin as Camunda Platform / C7 only

__Why should we do it?__

The plugin should work smoothly alongside our newest changes.

### LINKED ISSUE #20: Remove menu folder and reference
**What should we do?**

Remove the folder and the reference in the plugin entry point

**Why should we do this?**

Because it's unused.

### LINKED ISSUE #21: Plugin is not compatible with 5.0.0
**Issue**

Plugin is not compatabile with 5.0.0

**How to reproduce**

Clone this repo, move into `plugins` of `camunda-modeler` and start camunda modeler.

Now this happens:

Screenshot_20220317_094255

### PR DESCRIPTION
**Pull request by Skaiir**
* migrated to latest properties panel
* cleaned up repo

Closes #17
Closes #20
Closes #21

### CODE REVIEW

**Comment by Skaiir** [comment]
Somehow managed to merge this one manually without breaking everything.

**Comment by barmac** [comment]
Let's update the compatibility table as well: https://github.com/camunda/camunda-modeler-process-io-specification-plugin/tree/migrate-to-latest-properties-panel#compatibility-notice

**Comment by barmac** [comment]
> Merge branch 'master' into migrate-to-latest-properties-panel

How about if we just rebase and have no merge commits in the PR?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,17 @@ All notable changes to [camunda-modeler-process-io-specification-plugin](https:/
 
 ## Unreleased
 
-___Note:__ Yet to be released changes appear here._
+**\_Note:** Yet to be released changes appear here.\_
```
Comment (barmac) [inline CHANGELOG.md:7]:
```suggestion
___Note:__ Yet to be released changes appear here._
```

Let's not change styles.

**Review by barmac — CHANGES_REQUESTED** [review]

**Comment by Skaiir** [comment]
This is followed up in #23 since I can't squash a merge commit out of the git history.

**Pull request closed without merging**